test/rdoc/test_rdoc_parser_ruby.rb

```diff
@@ -4297,4 +4297,39 @@ class D
     assert_equal 'A::D', a_d.full_name
   end
 
+  COERCIBLE_OPERATORS = %w[ ! != !~ % & * ** + - / < << <= <=> == === =~ > >= >> ^ | ~ ]
```

Comment:
You can just use `RDoc::Parser::Ruby::OPERATOR_METHOD_NAMES` instead.

Reply:
Thanks for 'NAMES';  I had violated my own rule for distinguishing between string names and other objects.

Reply:
I mean you can delete this line since it's just a duplicate of `RDoc::Parser::Ruby::OPERATOR_METHOD_NAMES`.

Reply:
I want to be able to check whether the method and its test agree on the names.  See the first assertion.

Reply:
But that doesn't really test anything. A test is supposed to execute some code and check that its output matches what you expect. Checking that a constant is equal to a value isn't really testing anything meaningful (it's like testing that `assert_equal(1, 1)`).

Reply:
If I add a new binary op method `@` and add its name to the constant, the test will fail.  That's what I want to check.

Reply:
But they could just as easily also add it to this test and make it pass? This kind of things should be caught in reviews instead.

Reply:
I don't agree.  My point is to discover the error if they don't add it to the test.  Do you think that including this assertion is harmful?  If not, I'd like to retain it.

Reply:
I still strongly believe that there's no point in testing constants, and the best way to prevent incorrect changes to these constants is by making it easy to understand for other developers through proper naming and comments, and having PR reviews. But I don't think this is a particularly productive discussion to have, so I'm ok with keeping it.

Reply:
I will remove the test for the constant.  Should the test have its own constant, or should it rely on the one in the library?